docs/source/model_design/vivarium_features/cause_of_death_selection/index.rst

```diff
@@ -0,0 +1,216 @@
+..
+  Section title decorators for this document:
+  
+  ==============
+  Document Title
+  ==============
+  Section Level 1
+  ---------------
+  Section Level 2
+  +++++++++++++++
+  Section Level 3
+  ~~~~~~~~~~~~~~~
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _cause_of_death_selection:
+
+.. role:: underline
+    :class: underline
+
+=========================================================
+Cause of death selection
+=========================================================
+
+.. contents::
+   :local:
+
+Cause of death
+--------------
+
+When deaths are recorded on death certificates, a coroner typically records
+a **causal chain** of events, each causing the next, that directly caused the death.
+For epidemiological purposes, including in the GBD, it is usually the **underlying**
+cause that is most interesting: a **disease or injury** you find if you follow the causal chain
+back far enough.
+
+.. note::
+
+  Of course, you could imagine following the chain back even further,
+  for example saying that a lung cancer death was due to smoking or even that the smoking
+  was due to the lack of national tobacco control policies,
+  but this doesn't happen on death certificates.
+
+What does it mean when we say a death was *caused* by a disease?
+It means that in a counterfactual where the person didn't have that disease,
+they wouldn't have died.
+That explicit framing makes clear that there is some imprecision here:
+they wouldn't have died *when*?
+Of course, everyone is going to die eventually, regardless of what diseases they have.
+So that counterfactual can only delay their death by some finite amount of time.
+Perhaps they wouldn't have died that *minute* but they still would have died within
+the hour, or perhaps they would have lived 50 more years!
+
+Cause of death in GBD
+---------------------
+
+The Global Burden of Disease study (GBD), informed primarily by death certificates
+and by methods that seek to approximate them such as verbal autopsy, **assigns every
+death a single cause.**
```

Comment:
```suggestion
death a single cause.** (Or, very occasionally, when the interplay between causes is too important to ignore, GBD constructs a new cause to get at this, such as cause A.1.1.2, "HIV and Multidrug-resistant Tuberculosis without extensive drug resistance".)
```

Reply:
I'm using "cause" in this sentence to refer to GBD causes already, not causes on death certificates. Maybe I should just clarify that.